### DISCUSSION TITLE
config file during installation

### DISCUSSION THREAD

**Discussion by danielfowler** [duncanmcclean/guest-entries#54] [Help] [answered]
According to docs: https://guest-entries.duncanmcclean.com/configuration

a `guest-entries.php` file should've been created in my `config` directory but I don't see any such file. I just created one and added my collection handles and it appears to be working.

**Comment by danielfowler** [comment]
Statamic 4.2.0

**Comment by duncanmcclean** [comment, marked as answer]
Did you run the vendor:publish command during installation?

https://guest-entries.duncanmcclean.com/installation

**Comment by danielfowler** [comment]
Nope. Just delete this discussion thread to save me the humiliation.

**Comment by danielfowler** [comment]
In my defense, I was following the installation instructions from marketplace, not your docs. https://statamic.com/addons/duncanmcclean/guest-entries
